sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -109,11 +111,20 @@ def __str__(self):
         return str(self.message)
 
     def _build_message(self, body):
+        def _validate_body_type(body, allow_none=False):
+            if not (isinstance(body, (six.string_types, six.binary_type)) or (body is None and allow_none)):
+                raise TypeError("ServiceBusMessage body must be a string, bytes, None, or a list of string or bytes "
```

Comment:
same comment as the changelog one..
not mentioning "or a list of string or bytes element" would be my preference.

Reply:
As above, agreed, good call.